packages/playwright-core/src/client/network.ts

```diff
@@ -314,15 +334,21 @@ export class Route extends ChannelOwner<channels.RouteChannel> implements api.Ro
     }));
   }
 
-  async continue(options: { url?: string, method?: string, headers?: Headers, postData?: string | Buffer } = {}) {
-    await this._continue(options);
+  async continue(options: OverridesForContinue = {}) {
+    if (!this._routeChain)
+      throw new Error('Route is already handled!');
+    this._pendingContinueOverrides = { ...this._pendingContinueOverrides, ...options };
```

Comment:
Is chaining the continue options an expected feature? I don't see where the the overrides get passed from the last route nor `continue` can be called multiple times in a route. 👀